web/clusters.go

```diff
@@ -13,6 +13,7 @@ import (
 
 type Cluster struct {
 	Name string
+	Id   string
```

Comment:
I don't know if we use this `Cluster` and `ClusterList` types for something. Maybe it is just an old not removed data.
If this is the case we could remove them

Reply:
I 've check and yes, I auto-edited this but it's not really used anyway